### PR TITLE
Adds navigation

### LINKED ISSUE #34: Add navigation
Since we now have two separate pages (and will probably add another in the future), there should be some sort of navigation menu.

IMO it should be small (like a little hamburger button or something) so that it doesn't clutter the interface too much

### PR DESCRIPTION
**Pull request by asbjornh**
Fixes #34 

Also replaces button texts on audio controls with icons

### CODE REVIEW

**Comment by asbjornh** [comment]
If I understand correctly, this isn't mergable in its current state because of the client side routing that was added. I think we should revert the changes and do a separate PR for client side routing that fixes the issues with the firebase listeners

**Comment by pellebjerkestrand** [comment]
> > Yeah, I found the bug right after I committed. I would rather fix it now than revert and merge.
> 
> If you're able to do it today then I'm ok with that. If not I'll do the reverting and add a new issue :)

Boom.

File: app/source/components/notifications/notifications.jsx [hunk 1]
```diff
@@ -59,6 +59,14 @@ class Notifications extends React.Component {
     });
   }
 
+  componentWillUnmount() {
+    if (!this.unsubscribe) {
```
Comment (asbjornh) [inline app/source/components/notifications/notifications.jsx]: Will this ever happen?

File: app/source/js/firebase-helper.js [hunk 2]
```diff
@@ -8,19 +8,30 @@ firebase.initializeApp(firebaseInit);
...
+  const reference = database.ref('notifications');
+
+  reference
     .orderByChild('time')
     .limitToFirst(30)
     .on('value', snapshot => {
       callback(firebaseToArray(snapshot));
     });
+
+  return () => {
```
Comment (asbjornh) [inline app/source/js/firebase-helper.js]: Since `reference.off` is already a function, is it necessary to wrap it?

Comment (pellebjerkestrand) [reply app/source/js/firebase-helper.js]: Yeah, it did not work to return `reference.off` and call it from the component.

File: app/source/components/notifications/notifications.jsx (hunk 1 above)
Comment (pellebjerkestrand) [reply app/source/components/notifications/notifications.jsx]: Probably not, but I'm so very defensive 😅.

File: app/source/js/firebase-helper.js (hunk 2 above)
Comment (asbjornh) [reply app/source/js/firebase-helper.js]: Maybe `NOTE` it then? :)

Comment (pellebjerkestrand) [reply app/source/js/firebase-helper.js]: Hmm... maybe. IMO it was better than to start `bind`ing.

File: app/source/components/notifications/notifications.jsx (hunk 1 above)
Comment (asbjornh) [reply app/source/components/notifications/notifications.jsx]: This is your chance to live a little on the wild side!

**Comment by asbjornh** [comment]
💯

**Comment by asbjornh** [comment]
Just gonna merge this now since it's getting unclear who should review it :D

**Pull request merged**